runtime/jcl/module.xml

```diff
@@ -142,8 +142,21 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			<group name="se20">
 				<include-if condition="spec.java20"/>
 			</group>
-			<group name="jfr">
+			<group name="jfr_jdk11">
+				<include-if condition="spec.flags.opt_jfr"/>
+				<include-if condition="spec.java11"/>
+			</group>
```

Comment:
This is not consistent with the cmake version (`spec.java11` means `JAVA_SPEC_VERSION >= 11`); an exclude is needed:
```
				<exclude-if condition="spec.java12"/>
```
Similarly for the jdk17 and jdk21 groups.

Please also remove the unwanted space before `>` in `runtime/jcl/uma/jfr_objects.xml`:
```
	<object name="jdk_jfr_internal_JVM">
```

Reply:
For JDK23, should I keep it as is?